client/components/quizSingle/styles.js

```diff
@@ -0,0 +1,178 @@
+import styled from "styled-components";
+import { breakpoint } from "../../frontend-config";
+import { PrimaryButton, riseUp } from "../shared/styles";
+
+export const QuizHeader = styled.section`
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin: 0;
+  background: ${props => props.theme.colors.grey};
+`;
+
+export const QuestionHeaderContainer = styled.section`
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  width: 90%;
+  margin: 10px auto 40px auto;
+  transition-duration: 350ms;
+  animation: ${riseUp} 400ms ease-in-out;
+  animation-fill-mode: forwards;
+  animation-delay: ${props => props.animationDelay};
+
+  @media (min-width: ${breakpoint("md")}) {
+    max-width: 62%;
```

Comment:
We probably shouldn't use % for max-width in this case.
According to the design, the container itself is _around_ 850px

@cMikolai Any imputs for this? .. I don't know what the plan is for longer answers 🙂

Reply:
I agree, % here was not good solution, ill fix this.

Reply:
Fixed it using px value